src/tests/ftest/util/general_utils.py

```diff
@@ -923,7 +928,7 @@ def create_directory(hosts, directory, timeout=15, verbose=True,
 
     """
     mkdir_command = "/usr/bin/mkdir -p {}".format(directory)
-    command = get_clush_command(hosts, args="-S -v", command=mkdir_command, command_sudo=sudo)
+    command = get_clush_command(hosts, args="-S -B -v", command=mkdir_command, command_sudo=sudo)
```

Comment:
While scaling for 2KECBs I encountered an issue while agent startup when it was unable to create dirs to be able to copy on all ECB in time. I tried increasing the timeout and that did not help either. But adding fanout to clush command helped instead. 
```suggestion
    command = get_clush_command(hosts, args="-S -B -v -f 1024", command=mkdir_command, command_sudo=sudo)
```
Or something like that.
Let me know what you think of adding this here in the build.
IMO if there is a way to add some higher fanout values at a single place to all clush commands in our infrastructure for Aurora, it will he helpful.

Reply:
Really these direct clush commands should be replaced with run_remote, then run_remote can increase the fanout for all clush commands.